src/OSPSuite.Presentation/DTO/SimulationOutputMappingDTO.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.ParameterIdentifications;
+using OSPSuite.Presentation.DTO.ParameterIdentifications;
+using OSPSuite.Utility.Validation;
+
+namespace OSPSuite.Presentation.DTO
+{
+   public class SimulationOutputMappingDTO : OutputMappingDTO
```

Comment:
This looks fine to me. I do not think we need to do all of this just to add or remove one rule however
Can you create an issue to investigate removing SimulationOutputMappingDTO (explaining why it was created) and assign it to me?

Reply:
@msevestre issue created:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1670